themes/default/content/blog/deploy-ai-ml-vercel-app/index.md

```diff
@@ -0,0 +1,190 @@
+---
+date: 2023-09-11
+title: "Deploying Your AI/ML Chatbot Frontend To Vercel Using Pulumi"
+allow_long_title: true
+meta_desc: "Deploying Your AI/ML Chatbot Frontend To Vercel Using Pulumi"
```

Comment:
It might be good to see if we could expand this a little bit (we're allowed up to 160 characters here)